docs/vertx-runtime/master.adoc

```diff
@@ -27,6 +27,11 @@ include::topics/configuring-vertx.adoc[leveloffset=+2]
 
 include::topics/dev-guide-mission-intro.adoc[leveloffset=+1]
 
+[[vertx-available-missions-and-boosters]]
+== {Vertx} - Available Missions and Boosters
```

Comment:
Replace _{Vertx} - Available_ with _{Vertx}--Available._ (You should use an em dash, which is represented by two hyphens in the source, instead of a hyphen surrounded by spaces.)